portal/server/utils/csv-utils.js

```diff
@@ -236,7 +236,11 @@ const extractCsvData = async (file) => {
       // Read the .xlsx file using exceljs
       const workbook = new ExcelJS.Workbook();
       await workbook.xlsx.load(file.buffer, { sheetStubs: true }).then(async () => {
-        const worksheet = workbook.worksheets[0]; // Assume the utilisation report data is on the first sheet
+        // An Excel file must contain at least 1 visible worksheet at all times.
```

Comment:
Please can we still also have the comment about the report data being on the first sheet just reworded:
`Assume the utilisation report data is on the first visible sheet.`

Reply:
Fair point - updated to this thanks

Also removed the comment that a `file must contain at least 1 visible worksheet at all times` as I feel the error message in the `if` block now covers that

Reply:
Added the comment back now thanks